app/src/main/java/com/kickstarter/ui/activities/compose/search/FilterMenuSheet.kt

```diff
@@ -157,6 +165,147 @@ fun FilterMenuSheet(
     }
 }
 
+@OptIn(ExperimentalLayoutApi::class)
+@Composable
+private fun OtherFiltersRow(
+    modifier: Modifier = Modifier,
+    text: String = stringResource(R.string.Show_only_fpo),
+    selectedStaffPicked: MutableState<Boolean> = mutableStateOf(false),
+    callbackStaffPicked: (Boolean?) -> Unit = {},
+    selectedStarred: MutableState<Boolean> = mutableStateOf(false),
+    callbackStarred: (Boolean?) -> Unit = {},
+    selectedSocial: MutableState<Boolean> = mutableStateOf(false),
+    callbackSocial: (Boolean?) -> Unit = {},
+    selectedRecommended: MutableState<Boolean> = mutableStateOf(false),
+    callbackRecommended: (Boolean?) -> Unit = {},
+) {
+    val backgroundDisabledColor = colors.backgroundDisabled
+    val dimensions: KSDimensions = KSTheme.dimensions
+
+    val currentStaffPicked = remember { selectedStaffPicked }
+    val currentStarred = remember { selectedStarred }
+    val currentSocial = remember { selectedSocial }
+    val currentRecommended = remember { selectedRecommended }
+
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .drawBehind {
+                drawLine(
+                    color = backgroundDisabledColor,
+                    start = Offset(0f, size.height),
+                    end = Offset(size.width, size.height),
+                    strokeWidth = dimensions.dividerThickness.toPx()
+                )
+            }
+            .padding(
+                start = dimensions.paddingLarge,
+                top = dimensions.paddingLarge,
+                bottom = dimensions.paddingLarge,
+                end = dimensions.paddingMediumSmall
+            )
+    ) {
+        Column(
+            modifier = Modifier.testTag(OTHERS_ROW)
+        ) {
+            Text(
+                text = text,
+                style = typographyV2.headingLG,
+                color = colors.textPrimary
+            )
+
+            Row(
+                modifier = Modifier.testTag(DiscoveryParams::recommended.name),
```

Comment:
Notice here every row has as testtag the name of the field from `DiscoveryParams` it represents